packages/webpack-extensive-lodash-replacement-plugin/index.js

```diff
@@ -332,74 +330,106 @@ const LODASH_MODULE_NAMES = [
 	'zipWith',
 ];
 
-function throwError( message, error = null ) {
-	throw new Error(
-		`[ExtensiveLodashReplacementPlugin] ${ message }${ error ? ' Error: ' : '' }${ error }`
+function createError( message, error = null ) {
+	return new Error(
+		`[ExtensiveLodashReplacementPlugin] ${ message }${ error ? ` Error: ${ error }` : '' }`
 	);
 }
 
+function getModuleForPath( moduleResolver, rootPath, packageName ) {
+	return new Promise( ( resolve, reject ) => {
+		moduleResolver.resolve( {}, rootPath, packageName, {}, ( error, filepath, context ) => {
```

Comment:
Webpack calls the callback parameters `resource` and `resourceResolveData` instead of `filepath` and `context`. The `context` name especially is not great. It's a generic word -- anything can be "context" 🙂

Reply:
Can't remember where I found that convention. I agree, it's bad naming :) Fixed!